stellarphot/core.py

```diff
@@ -438,6 +438,9 @@ def __init__(
                 **kwargs,
             )
 
+            # From this point forwarsd we should be using self to get at any data
```

Comment:
```suggestion
            # From this point forward we should be using self to get at any data
```
Same typo noted in #470

Reply:
Was fixed there